src/freenet/support/SortedIntSet.java

```diff
@@ -221,12 +191,11 @@ public synchronized void clear() {
 	 * @return sorted array of all items
 	 */
 	public synchronized int[] toArrayRaw() {
```

Comment:
Well, those plugins are under our control, I guess… use the other call in those plugins, deprecate it here, remove it once the new plugin versions are released. :)